### ISSUE TITLE
Issue on updating to 1.5.0

### ISSUE THREAD

**Issue by hzoo** [webpack/webpack-dev-middleware#58]
Getting an issue from 1.4.1 to 1.5.0 and seems to be from https://github.com/webpack/webpack-dev-middleware/pull/27/files but not sure if it's a package issue or on my side.

```
TypeError: res.send is not a function
    at processRequest (/app/node_modules/webpack-dev-middleware/middleware.js:189:8)
    at ready (/app/node_modules/webpack-dev-middleware/middleware.js:96:20)
    at webpackDevMiddleware (/app/node_modules/webpack-dev-middleware/middleware.js:162:3)
```

**Comment by F21** [comment]
Also experiencing this problem when trying to use the middleware with browsersync.

**Comment by kaicataldo** [comment]
More info:

`res.send()` is an Express method, while `res.end()` is a part of Node core. We're using [karma-webpack](https://github.com/webpack/karma-webpack) for our unit testing and are not using the [webpack-dev-server](https://github.com/webpack/webpack-dev-server) that the change pointed to by @hzoo seems assume we're using.

**Comment by garza** [comment]
Also hit this when running the browser sync starter recipes for web pack ([https://github.com/BrowserSync/recipes]).  Reverting the change pointed to by @hzoo fixed the issue.

**Comment by hourliert** [comment]
Same problem here.

Stack:

```
/Users/thomashourlier/Projects/react/co-net/node_modules/webpack-dev-middleware/middleware.js:189
			res.send(content);
			    ^

TypeError: res.send is not a function
    at processRequest (/Users/thomashourlier/Projects/react/co-net/node_modules/webpack-dev-middleware/middleware.js:189:8)
    at continueBecauseBundleAvailible (/Users/thomashourlier/Projects/react/co-net/node_modules/webpack-dev-middleware/middleware.js:59:5)
    at Array.forEach (native)
    at /Users/thomashourlier/Projects/react/co-net/node_modules/webpack-dev-middleware/middleware.js:58:8
    at nextTickCallbackWith0Args (node.js:452:9)
    at process._tickDomainCallback (node.js:422:13)
```

**Comment by dlindahl** [comment]
Same but with hapi due to the same reason mentioned by [@kaicataldo](https://github.com/webpack/webpack-dev-middleware/issues/58#issuecomment-173749983)

**Comment by afc163** [comment]
+1

**Comment by mgcrea** [comment]
:+1:

**Comment by hzoo** [comment]
Fixed in 1.5.1 (https://github.com/webpack/webpack-dev-middleware/pull/59)
